fs/operations/operations.go

```diff
@@ -1301,6 +1301,14 @@ func Cat(ctx context.Context, f fs.Fs, w io.Writer, offset, count int64) error {
 			err = fs.CountError(err)
 			fs.Errorf(o, "Failed to send to output: %v", err)
 		}
+		if len(sep) >= 0 {
+			sepReader := bytes.NewReader(sep)
```

Comment:
This will do to write the sep to the stream, no need to make a reader.

   _ , err =  w.Write(sep)

Reply:
Ok, I was wondering about something like that, but wasn't sure if there was a reason to use `io.Copy()`. Updated, ran the test, and confirmed the behavior for my use case also!